### DISCUSSION TITLE
Vscode debug display

### DISCUSSION THREAD

**Discussion by ntm-dev** [microsoft/vscode-discussions#749] [Extension Show and Tell]
I don't know which updated version of vscode or php xdebug shows below:

![image](https://github.com/microsoft/vscode-discussions/assets/27183035/e18707eb-db5d-4d56-ae76-2194fee30601)

However, it only shows up on my Macpro M1 computer.
On Mac mini M1 it doesn't show up.

Both computers run the following versions:
Vscode:  1.81.0
PHP Debug: 1.32.1

Can anyone help me display it on mac mini.
Thank for your help

**Comment by gjsjohnmurray** [comment]
Please check your `debug.inlineValues` setting.

Please also note that this vscode-discussions forum is currently intended for people who are **developing** extensions for VS Code. More information including pointers to other sites are available [here](https://github.com/microsoft/vscode-discussions/discussions/5).

**Reply by ntm-dev** [reply]
hi [gjsjohnmurray](https://github.com/gjsjohnmurray),

Thank for your support.

As your answer, I enabled `debug.inlineValues`.
But it still doesn't work.

_I understand this vscode-discussions forum is currently intended for people who are developing extensions for VS Code.
so I'll take note for next time_

Thank for your support again.

**Comment by RanaRabees** [comment]
The reason why the PHP Debug window is not showing up on your Mac mini M1 is most likely because Xdebug is not enabled. To enable Xdebug, you need to add the following lines to your php.ini file:

```
xdebug.remote_enable=1
xdebug.remote_autostart=1
xdebug.remote_port=9003
```

Once you have added these lines to your php.ini file, you need to restart your web server. Then, you should be able to see the PHP Debug window in VS Code.

Here are the steps on how to enable Xdebug on your Mac mini M1:

1. Open a terminal window.
2. Run the following command to find the location of your php.ini file:

```
php -i | grep php.ini
```

3. Open the php.ini file in a text editor.
4. Add the following lines to the end of the file:

```
xdebug.remote_enable=1
xdebug.remote_autostart=1
xdebug.remote_port=9003
```

5. Save the php.ini file.
6. Restart your web server.
7. Open VS Code and open the PHP file that you want to debug.
8. Put a breakpoint in the code.
9. Press `F5` to start debugging.

The PHP Debug window should now be visible in VS Code.

If you are still having trouble getting the PHP Debug window to show up, you can try the following:

* Make sure that you are using the latest versions of VS Code and the PHP Debug extension.
* Check the settings for the PHP Debug extension to make sure that they are correct.
* Try debugging a different PHP file.
* If you are still having trouble, you can ask for help in the VS Code community forum.
